tests/tests/test_mender_connect.py

```diff
@@ -180,3 +181,70 @@ def test_bogus_proto_message(
             assert prot.props["status"] == protomsg.PROP_STATUS_ERROR
             assert prot.protoType == 12345
             assert prot.typ == proto_shell.MSG_TYPE_SPAWN_SHELL
+
+    def test_session_recording(
+        self, class_persistent_standard_setup_one_client_bootstrapped
+    ):
+        def get_cmd(ws, timeout=1):
+            pmsg = protomsg.ProtoMsg(1)
+            body = b""
+            while True:
+                try:
+                    while True:
+                        msg = ws.recv(timeout)
+                        b = pmsg.decode(msg)
+                        if pmsg.typ == proto_shell.MSG_TYPE_SHELL_COMMAND:
+                            body += b
+                except TimeoutError:
+                    return body
+
+        session_id = ""
+        session_bytes = b""
+        with devconnect.get_websocket() as ws:
+            # Start shell.
+            shell = proto_shell.ProtoShell(ws)
+            body = shell.startShell()
+            assert shell.protomsg.props["status"] == protomsg.PROP_STATUS_NORMAL
+            assert body == b"Shell started"
```

Comment:
can we have "Shell started" as a const/var?

Reply:
Sure